Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.12-slim
+LABEL maintainer="testtest@gmail.com"
+ENV PYTHONUNBUFFERED=1
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
Ensure that the `.dockerignore` file is properly configured to exclude unnecessary files and directories from being copied into the Docker image. This will help in reducing the image size and improving build efficiency.